src/NuGet.Core/NuGet.Configuration/Resources.resx

```diff
@@ -315,4 +315,19 @@
     <value>Allowed value(s): {0}</value>
     <comment>{0} - list of allowed values for an attribute</comment>
   </data>
+  <data name="Error_AtLeastOneSourcePerPackageMode" xml:space="preserve">
+    <value>{0}: Package namespace for '{1}' is not listed on any source. Any package id must be in one or more matching namespace declarations.</value>
```

Comment:
Note that if you move this logic to the restore command instead, you'll only need to create a RestoreLogMessage and you won't need to format the NU code.